ui-v2/app/styles/components/action-group/skin.scss

```diff
@@ -0,0 +1,40 @@
+.action-group {
+  @extend %action-group;
+}
+%action-group label {
+  border-radius: $radius-small;
+  cursor: pointer;
+}
+%action-group ul {
+  border: $decor-border-100;
+  border-radius: $radius-small;
+  box-shadow: 0 2px 7px rgba(0, 0, 0, 0.21);
```

Comment:
> Will box-shadows eventually become variables as well?

Yes, good you think this should happen 👍 

> Why not make this variables now along with everything else?

Mainly because I'm concentrating on colors for the moment, as I suggested a couple of months back. The box shadows 'require' colors, so it would be best to get the colors nailed first if possible. I'll probably move on to these pretty soon though I was chatting to @opihana about it yesterday.